src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Ionicons } from '@expo/vector-icons';
+import { FlatList, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { profileSectionData } from 'src/utils/profileSectionData';
+import { Button, ProfileHeader, Text } from 'src/components';
+import { ScreenTemplate } from 'src/templates';
+import { useTheme } from 'src/theme';
+
+const ProfileScreen = () => {
+  const isLoggedIn = true;
+  const { setScheme, isDark } = useTheme();
+
+  const renderSectionItem = ({ item }: any) => (
+    <TouchableOpacity style={styles.sectionWrapper}>
+      <View style={styles.sectionLeftWrapper}>
+        <Ionicons name={item.icon} size={26} />
+        <Text>{item.title}</Text>
+      </View>
+      <Ionicons name="chevron-forward-outline" size={22} />
+    </TouchableOpacity>
+  );
+
+  const changeTheme = () => {
+    if (isDark) {
+      setScheme('light');
+    } else {
+      setScheme('dark');
+    }
+  };
+
+  return (
+    <ScreenTemplate>
+      <FlatList
+        ListHeaderComponent={<ProfileHeader isLoggedIn={isLoggedIn} />}
+        data={[
+          {
+            label: 'Settings',
+            data: isLoggedIn ? profileSectionData.whenLoggedIn : profileSectionData.whenLoggedOut,
+          },
+          { label: 'Help', data: profileSectionData.help },
+        ]}
+        keyExtractor={(item, index) => index.toString()}
+        renderItem={({ item }) => (
```

Comment:
`renderItem` and `ListFooterComponent` should be a separate function or component and should be used like `ListHeaderComponent`